schemas/RecipeFlow.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/RecipeFlow.json",
+    "title": "RecipeFlow",
+    "description": "The linkage between a recipe process, an action that structures a recipe, and a resource specification.",
+    "type": "object",
+    "properties": {
+        "name": {
+            "description": "An informal or formal textual identifier for the item. Does not imply uniqueness.",
+            "type": "string"
+        },
+        "action": {
+            "description": "Relates a process input or output to a verb, such as consume, produce, work, modify, etc.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Action.json"
+        },
+        "resourceQuantity": {
+            "description": "The amount and unit of the economic resource counted or inventoried.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/QuantityValue.json"
+        },
+        "effortQuantity": {
+            "description": "The amount and unit of the work or use or citation effort-based action. This is often a time duration, but also could be cycle counts or other measures of effort or usefulness.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/QuantityValue.json"
+        },
+        "recipeFlowResource": {
+            "description": "The resource definition referenced by this flow in the recipe.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/RecipeResource.json"
+        },
+        "recipeInputOf": {
```

Comment:
This and the next attribute name are not self-defining. They mean input output of a recipeProcess, but only the description says so.